vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/TouchSteps.java

```diff
@@ -140,7 +140,7 @@ public void swipeToElement(SwipeDirection direction, Locator locator, Duration s
         }
     }
 
-    private Rectangle createSwipeArea()
+    private Rectangle getSearchContextRectangle()
```

Comment:
does it make sense to inline this method?

Reply:
not now, it should be moved to UiContext